modules/util/portal-tools-soy-builder/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -5,6 +5,7 @@
 		<pluginExecution>
 			<pluginExecutionFilter>
 				<goals>
+					<goal>build-soy</goal>
```

Comment:
You're right! Also, I closed the pull because I want to figure out a way to reuse `portal-tools-soy-builder` in `gradle-plugins-soy` without adding all those transitive depencencies to Gradle...